File: epi_judge_java/epi/CountInversions.java
package epi;

import epi.test_framework.EpiTest;
import epi.test_framework.GenericTestHandler;

import java.util.List;

public class CountInversions {
  @EpiTest(testfile = "count_inversions.tsv")

  public static int countInversions(List<Integer> A) {
    // Implement this placeholder.
    return 0;
  }

  public static void main(String[] args) {
    GenericTestHandler.executeTestsByAnnotation(
        new Object() {}.getClass().getEnclosingClass(), args);
  }
}

File: epi_judge_cpp/count_inversions.cc
#include <vector>
#include "test_framework/generic_test.h"

using std::vector;

int CountInversions(vector<int> A) {
  // Implement this placeholder.
  return 0;
}

int main(int argc, char* argv[]) {
  std::vector<std::string> args{argv + 1, argv + argc};
  std::vector<std::string> param_names{"A"};
  return GenericTestMain(args, "count_inversions.tsv", &CountInversions,
                         DefaultComparator{}, param_names);
}
